packages/core/src/configuration.ts

```diff
@@ -144,6 +162,27 @@ export function buildConfiguration(userConfiguration: UserConfiguration, buildEn
     }
   }
 
+  if (transportConfiguration.buildMode === 'staging') {
```

Comment:
You shouldn't we have an enum out of `export type BuildMode = 'release' | 'staging' | 'e2e-test'` instead of having twice the same 'staging' ?

Reply:
it is not an enum but let's use some for this kind of type, it would also avoid the casting